design/one-pager-composition-environment-beta.md

```diff
@@ -0,0 +1,378 @@
+# Beta Composition Environment
+
+- Owner: Philippe Scorsolini (@phisco)
+- Reviewers: TBD
+- Status: Draft
+
+# Background
+
+The "Composition Environment" concept was introduced in the original
```

Comment:
I'll move the old one to defunct once this one gets approved 👍

Reply:
Should we move it to defunct right away or when this one is "implemented"? 

Based on our [conventions](https://github.com/crossplane/crossplane/blob/a53d5f1ff9e0f50e44b421ce5a811251a314437c/design/README.md), I believe the old one should stay `Accepted` with a note that this one supersedes it until its fully implemented, then it should go to `Defunct`? I think this would go into the "semi-defunct" category described at the bottom.

Reply:
yes, I think it makes sense to keep it as semi-defunct, as the parts about `EnvironmentConfigs` will still apply. I'll see what we did in the past and for similar cases and do the same.